deploy/PortabilityService.ServiceFabricMeshApplication/Reference/Deploy.md

```diff
@@ -0,0 +1,16 @@
+Manual SeaBreeze Deployment Instructions
```

Comment:
Should this be "mesh"?

Reply:
Good catch; forgot to update this one :)